src/components/MoviesList/MoviesList.tsx

```diff
@@ -4,12 +4,17 @@ import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  filter: string;
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
+export const MoviesList: React.FC<Props> = ({ movies, filter }) => (
   <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
+    {movies.filter(element => element.title.toLocaleLowerCase()
+      .includes(filter.toLocaleLowerCase())
+    || element.description.toLocaleLowerCase()
+      .includes(filter.toLocaleLowerCase()))
+      .map(movie => (
+        <MovieCard key={movie.imdbId} movie={movie} />
+      ))}
```

Comment:
lets destructure movie and you can put it to separate function to no rewrite logic (you do .includes(filter.toLocaleLowerCase() twice)